File: app/src/main/java/com/example/maunorafiq/pawangcuaca/presentation/forecast/ForecastView.java
package com.example.maunorafiq.pawangcuaca.presentation.forecast;

import com.example.maunorafiq.pawangcuaca.presentation.model.ForecastModel;
import com.example.maunorafiq.pawangcuaca.presentation.model.WeatherModel;
import com.example.maunorafiq.pawangcuaca.presentation.base.LoadDataView;

/**
 * Created by maunorafiq on 11/29/16.
 */

public interface ForecastView extends LoadDataView {

    void renderForecastWeather (ForecastModel forecastWeather);

    void renderForecastWeather (WeatherModel weatherModel);
}

File: app/src/main/java/com/example/maunorafiq/pawangcuaca/presentation/forecast/ForecastView.kt
package com.example.maunorafiq.pawangcuaca.presentation.forecast

import com.example.maunorafiq.pawangcuaca.presentation.base.LoadDataView
import com.example.maunorafiq.pawangcuaca.presentation.model.CityModel
import com.example.maunorafiq.pawangcuaca.presentation.model.ForecastModel
import com.example.maunorafiq.pawangcuaca.presentation.model.WeatherModel

/**
 * Created by maunorafiq on 11/29/16.
 */

interface ForecastView : LoadDataView {

    fun renderForecastWeather(forecastModel: ForecastModel)

    fun renderForecastWeather(weatherModel: WeatherModel)
}
